### DISCUSSION TITLE
Mark matched term of group

### DISCUSSION THREAD

**Discussion by jelly-garden** [sa-si-dev/virtual-select#113] [Q&A] [answered]
Can I mark matched term of group?

I made options.
```
const _options = this.groupList.map((group) => ({
	label: group['group_name'],
	value: group['group_code'],
	options: group['users'].map((user) => ({
		label: `${user['user_name']} (${user['user_account']})`,
		value: user['uid'],
		alias: [group['group_name']]
	}))
}));
```
And turn on the `markSearchResults` property
```
VirtualSelect.init({
	...
	markSearchResults: true
});
```

Is there any way to mark matched term of group like this?
![image](https://user-images.githubusercontent.com/60183166/156514337-efa09415-f093-46e4-acee-f95fc20e0714.png)

**Comment by sa-si-dev** [comment]
As of now, searching will compare the search value with option's label.
I will add support to include group names in the search case in the future release.

**Comment by sa-si-dev** [comment, marked as answer]
@jelly-garden This feature has been implemented as `searchGroup: true` property in the [release v1.0.28](https://github.com/sa-si-dev/virtual-select/releases/tag/v1.0.28)
